collectors/health.go

```diff
@@ -132,6 +135,15 @@ type ClusterHealthCollector struct {
 
 	// ClientIOWriteOps shows the rate of total write operations conducted by all clients on the cluster.
 	ClientIOWriteOps prometheus.Gauge
+
+	//CacheFlushIORate shows the i/o rate at which data is being flushed from the cache pool
```

Comment:
Nitpicks:
- Space after `//`, like `// CacheFlushIORate`
- Add a period.

Same for the other two new metrics.  For consistency with the other code.